packages/cdktf-cli/templates/typescript/help

```diff
@@ -23,7 +23,6 @@
 
 
  Upgrades:
-   npm run get           Import/update Terraform providers and modules (you should check-in this directory)
```

Comment:
Why remove this? While it may not be needed for many users, seems like a useful shortcut.

Reply:
Was lost while copying `typescript-minimal` template. added it again.